Pack-Smart/pack-smart-fe
# Refactor/burger

## What's this PR do?
- Adds handler functions to burger

## Where should the reviewer start?
- `Hamburger.js`

## How should this be manually tested?
- Test in browser in mobile mode. Burger menu should default to close and should close after a link is clicked. 

## Any background context you want to provide?
- n/a

## What are the relevant tickets?
- closes #72

## Screenshots/GIFs
- <img width="444" alt="Screen Shot 2021-02-27 at 4 52 23 PM" src="https://user-images.githubusercontent.com/68157759/109401562-72aec180-791d-11eb-8eca-b6fde25a394a.png">

## Questions:
- no

## Did you update the README?
- what do you think?
